packages/core/js/colors.2020.js

```diff
@@ -0,0 +1,79 @@
+export const blue = {
+  base: '#0084BD',
+  tint1: '#69CAFA',
+  tint2: '#005880'
+}
+export const teal = {
+  base: '#00BAA5',
+  tint1: '#84F0E1',
+  tint2: '#007D6F'
+}
+export const green = {
+  base: '#009E52',
+  tint1: '#6EDBA6',
+  tint2: '#006133'
+}
+export const lime = {
+  base: '#8CCA08',
+  tint1: '#CDF084',
+  tint2: '#5C8D00'
+}
+export const yellow = {
+  base: '#FAD000',
+  tint1: '#FFF480',
+  tint2: '#BD9400'
+}
+export const orange = {
+  base: '#F59127',
+  tint1: '#FFBA70',
+  tint2: '#BA6409'
+}
+export const red = {
+  base: '#D71D09',
+  tint1: '#FC6F5F',
+  tint2: '#950E00'
+}
+export const pink = {
+  base: '#E7558B',
+  tint1: '#FE9ABF',
+  tint2: '#AC3661'
+}
+export const purple = {
+  base: '#8359DF',
+  tint1: '#B899FA',
+  tint2: '#5834A7'
+}
+
+export const dark = {
+  elevation1: '#0D0F12',
+  elevation2: '#181C20',
+  elevation3: '#1E2429',
+}
+
+export const light = {
+  elevation1: '#F4F5F7',
+  elevation2: '#FFFFFF'
+}
+export const utility = {
+  40: 'rgba(117, 135, 153, 0.4)',
+  20: 'rgba(117, 135, 153, 0.2)'
+}
+
+export const contrast = {
+  textIconDarkBgHigh: 'rgba(255, 255, 255, 0.95)',
+  textIconDarkBgLow: 'rgba(255, 255, 255, 0.65)',
+  textIconLightBgHigh: 'rgba(0, 0, 0, 0.95)',
+  texIconLightBgLow: 'rgba(0, 0, 0, 0.55)'
+}
+
+export const brandGradientSkills = {
+  background: 'linear-gradient(90deg, #E80A89 0%, #F15B2A 100%)',
+  stop0: '#E80A89',
+  stop100: '#F15B2A'
+}
+
+export const brandGradientFlow = {
+  background: 'linear-gradient(90deg, #2968B2 0%, #27AAE1 100%)',
+  stop0: '#2968B2',
+  stop100: '#27AAE1'
+}
```

Comment:
Is what's in the airtable:

https://airtable.com/shr8lh2JLurHl19uL/tbl1ucbCYnUmu611s?blocks=hide

meant to be in this file?  It doesn't seem to be 1:1.

For instance, airtable doesn't have "utility" or "contrast" categories.  What, then, do those map to?

Reply:
I'll add the other values from airtable but background is a shortcut I think wee need to keep since that's that usual application.